src/Menu/menuUtils.spec.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env mocha */
+import {assert} from 'chai';
+import * as Utils from './menuUtils';
+
+describe('Menu Utils', () => {
+  describe('HotKeyHolder', () => {
+    let hotKeyHolder;
+    beforeEach(() => {
+      hotKeyHolder = new Utils.HotKeyHolder();
+    });
+    afterEach(() => {
+      hotKeyHolder = null;
+    });
+    it('returns the key appended', () => {
+      assert.equal(hotKeyHolder.append('k'), 'k');
+    });
+    it('holds keys within 500ms and dispose these afterwards', () => {
+      hotKeyHolder.append('k');
+      return timeout(100)
+        .then(() => {
+          assert.equal(hotKeyHolder.append('o'), 'ko');
+          assert.equal(hotKeyHolder.append('k'), 'kok');
+          assert.equal(hotKeyHolder.append('o'), 'koko');
+          assert.equal(hotKeyHolder.append('s'), 'kokos');
+        })
+        .then(() => timeout(400))
+        .then(() => {
+          assert.equal(hotKeyHolder.append('a'), 'kokosa');
+          assert.equal(hotKeyHolder.append('k'), 'kokosak');
+          assert.equal(hotKeyHolder.append('e'), 'kokosake');
+        })
+        .then(() => timeout(600))
+        .then(() => {
+          assert.isNull(hotKeyHolder.lastKeys);
```

Comment:
Actually I prefer that way too, but in my opinion as long as we depend on `chai`, we should use the full of API. If we want simpler one,  we can discuss to stop using `chai`.